doc/source/scalability/innodb_adaptive_hash_index_partitions.rst

```diff
@@ -4,7 +4,7 @@
  Multiple Adaptive Hash Search Partitions
 ==========================================
 
-The |InnoDB| adaptive hash index can have contention issues on multi-core systems when you run a mix of read and write queries that need to scan secondary indexes. This feature splits the adaptive hash index across several partitions to avoid such problems.
+The |InnoDB| adaptive hash index can have contention issues on multi-core systems when you run a mix of read and write queries that need to scan secondary indexes. This feature splits the adaptive hash index across several equal-sized partitions to avoid such problems.
```

Comment:
more details were mentioned in the second paragraph that wasn't shown in the diff. I've expanded it as well to mention the bug comment.